src/app/rundown/components/rundown/rundown.component.ts

```diff
@@ -16,7 +16,10 @@ export class RundownComponent implements OnInit, OnDestroy {
 
   public rundown?: Rundown
   public adLibPieceIdentifiers: Identifier[] = []
+  public isLoading: boolean = true
   private rundownSubscription?: SubscriptionLike
+  private isLoadingSubscription?: SubscriptionLike
+  private subscriptions: SubscriptionLike[] = []
```

Comment:
This isn't used and should be deleted.

Reply:
By "this" I mean the `subscriptions` variable.

Reply:
Good catch, a left-over from a different implementation i tried. I'll remove it!